src/Nethermind/Nethermind.Evm/Witness/VerkleExecWitness.cs

```diff
@@ -88,26 +78,22 @@ public bool AccessForTransaction(Address originAddress, Address? destinationAddr
     /// <returns></returns>
     public bool AccessForGasBeneficiary(Address gasBeneficiary, ref long gasAvailable)
     {
-        if (!AccessVersion(gasBeneficiary, ref gasAvailable)) return false;
-        if (!AccessNonce(gasBeneficiary, ref gasAvailable)) return false;
-        if (!AccessBalance(gasBeneficiary, ref gasAvailable)) return false;
+        if (!AccessBasicData(gasBeneficiary, ref gasAvailable)) return false;
```

Comment:
this wont make a differene because we dont charge gas for these -> but add isWrite: true for basic data